### ISSUE TITLE
Vertical USB

### ISSUE THREAD

**Issue by rusefillc** [rusefi/alphax-4chan#14]
Considering case layout vertical would work better

![20220330_233408](https://user-images.githubusercontent.com/48498823/160971192-4fd593b1-9f36-4df1-a6a6-e1eef114210d.jpg)

**Comment by rusefillc** [comment]
@ElDominio what do you think?

Vertical usb is a bit more fragile, probably no space for usb-b?

**Comment by ElDominio** [comment]
While I understand, I typically drill a hole for people demanding access, it can be done but I don't know if it's worth it.

USB-B is the printer one right?

**Comment by rusefillc** [comment]
yep, USB-B is the printer one

vertical connector is about 14x14mm which we do not have

**Comment by ElDominio** [comment]
Yeah, if we can switch to type B vertical, it could be done, but if we keep miniusb I'd say keep horizontal.

**Comment by andreika-git** [comment]
Unfortunately, it's highly unlikely that we can find more space to fit the USB-B-Vertical connector without total rearrangement of the whole board... Unless we remove the left mounting hole (near the bluetooth module)...
